src/control/dmg/main.go

```diff
@@ -51,6 +56,19 @@ var (
 	config = client.NewConfiguration()
 )
 
+func loadCredentials() (credentials.TransportCredentials, error) {
+	if opts.Insecure {
+		fmt.Println("Using Insecure connection")
+		return nil, nil
+	}
+	caRootPath := config.CACert
```

Comment:
why create local variables here rather than just passing to method directly?

Reply:
It was originally a more complex statement. I just kept it this way to save some characters in the line below. I can pass them directly instead.